composer.json

```diff
@@ -17,8 +17,15 @@
         "php": ">=5.3.0",
         "lib-pcre": ">=4.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "4.8.*"
+    },
     "autoload": {
-        "psr-0": { "ML\\IRI": "" }
+        "classmap": ["src/"],
+        "psr-4": {
+            "ML\\IRI\\": "src/",
```

Comment:
Any reason to not simply point this to `""`, get rid of the second declaration and leave the `IRI.php` file where it currently is?

Reply:
Fair point @lanthaler, this is the standard I've used and I get scared to deviate but it doesn't seem like its necessary.